_episodes/14-modules.md

```diff
@@ -148,7 +148,7 @@ it added a directory to the beginning of our `$PATH`. Let's examine what's there
 ```
 {: .output}
 
-Taking this to it's conclusion, `module load` will add software to your `$PATH`. It "loads"
+Taking this to its conclusion, `module load` will add software to your `$PATH`. It "loads"
```

Comment:
This change should be on a different branch to contribute upstream

Reply:
I will create a branch `mattgillucl-hpc-typos` direct from the original repo. And put my typos corrected in there.

Reply:
Removed from this branch